data/translations.de.json

```diff
@@ -3,13 +3,13 @@
     "AcknowledgmentsItemsTitle": "Danksagung",
     "AcknowledgmentsItemsDescription": "Dient der Anerkennung von Beiträgen durch Beschreibung der Personen, die beigetragen haben.",
     "AcknowledgmentsItemsNamesTitle": "Liste der anzuerkennenden Namen",
-    "AcknowledgmentsItemsNamesDescription": "Enthält die Namen der anzuerkennenden Personen.",
-    "AcknowledgmentsItemsNamesItemsTitle": "Name der anzuerkennenden Person",
+    "AcknowledgmentsItemsNamesDescription": "Enthält die Namen der anzuerkennenden Entitäten.",
+    "AcknowledgmentsItemsNamesItemsTitle": "Name der anzuerkennenden Entität",
     "AcknowledgmentsItemsNamesItemsDescription": "Enthält den Namen einer einzelnen Person.",
```

Comment:
The corrected English version talks about `contributor` (`Beitragende`) instead of `entities`.

Reply:
I updated the English version and German translation accordingly. Includes some minor changes for other keys as well, see https://github.com/secvisogram/secvisogram/pull/263/commits/726c75223e329ceb16e2b56278585ab1de96dd98